src/cloud/release-notes/mcp-release-notes.md

```diff
@@ -16,6 +16,10 @@ The `{{site.data.var.mcp}}` package uses the following version sequence: `<major
 {:.bs-callout-info}
 See [Apply patches]({{site.baseurl}}/cloud/project/project-patch.html) to learn how to apply Magento patches and hot fixes to your {{site.data.var.ece}} project.
 
+## v1.0.2
```

Comment:
We need to add other items from https://github.com/magento/magento-cloud-patches/pulls?q=is%3Apr+is%3Aclosed+label%3A%22Release%3A+1.0.2%22 (these are mostly for future and we can call them out as dependency updates). Also there is a bug which was fixed in https://github.com/magento/magento-cloud-patches/pull/19, which would be great to document too